crates/evm/src/executor/mod.rs

```diff
@@ -408,7 +404,7 @@ impl Executor {
 
         let result = match &out {
             Some(Output::Create(data, _)) => data.to_owned(),
-            _ => Bytes::default(),
+            _ => alloy_primitives::Bytes(Bytes::default()),
```

Comment:
Can unwrap

Reply:
Done in. https://github.com/foundry-rs/foundry/pull/5768/commits/91107449b7bbc8e19a70cdb9ffc9489732cb146a